.php-cs-fixer.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+;
+
+$config = new PhpCsFixer\Config();
```

Comment:
Not a big deal neither but in my experience introducing a CS fixer has always been quite noisy, might be worth extracting it to a different PR?

Reply:
it's still small enough, so I'll leave it. I'll keep it in mind for the next time. I only added it, so we actually start using some PHP 7.1 features